log/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/cosmos/cosmos-sdk/log
```

Comment:
Given that it is a small PR, could you add all the tooling too?

Reply:
will do

Reply:
I think I got it all

Reply:
Just one typo, and could you add it to dependabot?

Reply:
Should we make it `cosmossdk.io/log`? Why not just have this interface in core? I think there should be a core logging service

Reply:
I thought core should not containing any implementation.
If you had an interface in core and the implementation in logger that's a bit weird. Imo better to keep close the interface from its implementation for small packages like that.

Agrees that a vanity url is better however.

Reply:
I think a no-op implementation in core is fine. That's all this has right? I don't have a strong preference on whether it goes in core, but I do think modules will want a logging service so that should probably go in core right?

Reply:
Right, but maybe in that log package we can add a function for these logger too, so we don't duplicate other loggers:
- https://github.com/cosmos/cosmos-sdk/blob/main/tools/cosmovisor/logger.go
- https://github.com/cosmos/cosmos-sdk/blob/main/tools/rosetta/lib/logger/logger.go
This was what makes the most sense to me, imo as this does not fit in core.

Reply:
No, that probably shouldn't be in core. So if we can have a simple interface with zero dependencies it can exist in core and log. Log can provide some implementations like zerolog and comet. And core can use the same interface. The main thing is I think getting rid of the dependency on comet as I mentioned here https://github.com/cosmos/cosmos-sdk/pull/14904#issuecomment-1421454385